Competition2018/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -116,14 +127,23 @@ public boolean isCalibrated() {
     public void setPower(double power) {
 
         if (contract.elevatorLowerLimitReady()) {
-            boolean sensorHit = calibrationSensor.get();
+            boolean sensorHit = lowerLimitSwitch.get();
             calibrationLatch.setValue(sensorHit);
 
             // If the lower-bound sensor is hit, then we need to prevent the mechanism from lowering any further.
             if (sensorHit) {
                 power = MathUtils.constrainDouble(power, 0, 1);
             }
         }
+        
+        if (contract.elevatorUpperLimitReady()) {
+            boolean sensorHit = upperLimitSwitch.get();
+            
+            //If the upper-bound sensor is hit, then we need to prevent the mechanism from rising any further.
+            if (sensorHit) {
```

Comment:
If the switch isn't available we _ignore the limits and drive the elevator anyway?_

Reply:
@WasabiFan if the sensor isn't installed (the only reason for contract.elevatorUpperLimitReady() to be false), then of course we allow the elevator to drive.  How could we react to a non-existent switch?

That being said, further down at https://github.com/Team488/TeamXbot2018/blob/6aae74082b644878f7cd51f46cb621cbb5563c2d/Competition2018/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java#L156 you can see that if the elevator is calibrated it will still stop at the "max height" via encoder.